src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -103,7 +104,7 @@ public async Task<List<string>> GetLatestManifestContentAsync(string packageId)
                 .FirstOrDefault();
 
             var packageContents = (await this.github.Repository.Content.GetAllContents(this.wingetRepoOwner, this.wingetRepo, version))
-                .Where(c => c.Type != ContentType.Dir);
+                .Where(c => c.Type != ContentType.Dir && Path.GetExtension(c.Name) == ".yaml");

```

Comment:
should be case-insensitive

Reply:
Changed to a case-insensitive string comparison

Reply:
We have an extension method for this already, EqualsIC